topics/proteomics/tutorials/metaquantome-function/tutorial.md

```diff
@@ -0,0 +1,424 @@
+---
+layout: tutorial_hands_on
+
+title: "metaQuantome 2: Function"
+zenodo_link: "https://doi.org/10.5281/zenodo.4110725"
+questions:
+ - "How do I look at differentially expressed proteins?"
+ - "How do I get top 5 or 10 function present in the sample?"
+ - "How can I visualize my results?"
+objectives:
+  - "A functional and quantitational analysis of metaproteomic mass spectrometry data."
+time_estimation: "1h"
+key_points:
+  - "With function and Quantitation inputs perform statistical analysis"
```

Comment:
can you make the keypoints more of a take-home message? So less of a "this is what we did", and more of a "this is an important conclusion or from this tutorial"